assignments/this.js

```diff
@@ -13,14 +13,62 @@
 
 // code example for Window Binding
 
+let sayMiddleName = function(){
+    console.log(this.middleName);
+};
+
+let scott = {
+    age: 23,
+    dateOfBirth: "09/25/1994",
+    middleName: "Michael",
+};
+
+window.middleName = "Michael";
+
+sayMiddleName();
+
+
 // Principle 2
 
 // code example for Implicit Binding
 
+let scott = {
```

Comment:
You should avoid declaring the same let variable multiple tines in the same scope.